blog/_posts/2020-08-12-building-root-with-distcc.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Building ROOT fast - distributed compilation with distcc"
+layout: archive
+author: Stefan Wunsch
+---
+
+Hello there, here is Stefan with a short guide on how you can compile ROOT distributed with distcc!
+
+ROOT is a huge project with more than 24 thousand files and above 4.6 million lines of code (counted with [`cloc`](https://github.com/AlDanial/cloc)). Consequently, a full rebuild from scratch takes a while! To give some numbers:
+
+```
+Building ROOT on 4 cores: 86 minutes
+```
+
+How to make this faster? Besides buying a more powerful computer, another answer from the software community is [`distcc`](https://distcc.github.io/). `distcc` allows to compile C, C++ and objective C(++) distributed, is simple to set up and already integrated into the ROOT build system!
```

Comment:
I think this is a good point to clarify an important pre-requisite: you need another machine ("distributed compilation" might not be familiar to some readers, but this should clarify what you are going to talk about).
Feel free to rephrase differently than how I suggest below, of course.

```suggestion
How to make this faster? Besides buying a more powerful computer, another answer from the software community is [`distcc`](https://distcc.github.io/). If you have a spare computer or a second computer reachable over the network, `distcc` allows to distribute compilation of C, C++ and objective C(++) code over multiple machines. It is simple to set up and has been recently integrated into the ROOT build system!
```

Reply:
I thought `distributed` tells already that there is more than one dedicated machine ;) But most likely you have a point! Never hurts to be more specific :)